src/main/webapp/components/sources-info-local/sources-info-local.js

```diff
@@ -0,0 +1,77 @@
+import React from 'react'
+
+import { useQuery } from '@apollo/react-hooks'
+import gql from 'graphql-tag'
+
+import StorageIcon from '@material-ui/icons/Storage'
+import OnlineIcon from '@material-ui/icons/CloudDoneOutlined'
+import OfflineIcon from '@material-ui/icons/OfflineBoltOutlined'
+import { SourcesInfo } from '../sources-info/sources-info'
+
+const sources = gql`
+  query SourcesInfoLocal($attribute: String!) {
+    facet(attribute: $attribute) {
+      value
+      count
+    }
+    sources {
+      isAvailable
+      sourceId
+      local
+    }
+  }
+`
+
+const pollInterval = gql`
+  query SourcePollInterval {
+    systemProperties {
+      sourcePollInterval
+    }
+  }
+`
+const FACET_ATTRIBUTE = 'ext.content-store-name'
+
+const useSourcePollInterval = init => {
+  const { data, loading, error } = useQuery(pollInterval)
+
+  if (loading || error) {
+    return init
+  }
+
+  return data.systemProperties.sourcePollInterval
+}
+
+export default () => {
+  const pollInterval = useSourcePollInterval(60000)
+
+  const { loading, data = {}, error, refetch } = useQuery(sources, {
+    pollInterval,
+    variables: { attribute: FACET_ATTRIBUTE },
+  })
+  let listOfsources = []
+  if (!loading) {
+    listOfsources = data.sources.map(source => {
+      source.Icon = source.isAvailable ? OnlineIcon : OfflineIcon
+      return source
+    })
+    const facetSources = data.facet
+    listOfsources = listOfsources.concat(
+      facetSources.map(facetSource => {
+        facetSource.Icon = StorageIcon
+        facetSource.isAvailable = true
+        return facetSource
+      })
+    )
+  }
+  if (error) {
+    return (
+      <SourcesInfo
+        loading={loading}
+        sources={sources}
+        error={error}
+        refetch={refetch}
+      />
+    )
+  }
+  return <SourcesInfo loading={loading} sources={listOfsources} />
```

Comment:
:pencil2: don't need the conditional - can just pass the error in directly
```suggestion
  return <SourcesInfo loading={loading} error={error} loading={loading} sources={listOfsources} />
```